### PR TITLE
[Merged by Bors] - feat: lift concrete description of colimits in Type to functor category

### PR DESCRIPTION
**Pull request by TwoFX**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Limits/FunctorToTypes.lean [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+universe w v₁ v₂ u₁ u₂
+
+variable {J : Type u₁} [Category.{v₁} J] {K : Type u₂} [Category.{v₂} K]
+
+variable (F : J ⥤ K ⥤ TypeMax.{u₁, w})
+
+theorem jointly_surjective (k : K) {t : Cocone F} (h : IsColimit t) (x : t.pt.obj k) :
+    ∃ j y, x = (t.ι.app j).app k y := by
+  let hev := PreservesColimit.preserves (F := (evaluation _ _).obj k) h
```
Comment (joelriou) [inline Mathlib/CategoryTheory/Limits/FunctorToTypes.lean:28]:
```suggestion
  let hev := isColimitOfPreserves ((evaluation _ _).obj k) h
```

**Comment by joelriou** [comment]
Thnks!

bord d+

**Comment by joelriou** [comment]
Thanks!

bors d+

**Comment by TwoFX** [comment]
bors d+

**Pull request closed without merging**